src/components/HeaderForm.js

```diff
@@ -948,25 +968,45 @@ class HeaderForm extends Component {
                   region={this.state.region}
                 />
               )}
+              
               {customFilesFlag && (
-                <div style={{ display: "flex" }}>
-                  {DataPositionFormRowComponent}
+                <div className="d-flex justify-content-between align-items-start">
+                  <div>
+                    {DataPositionFormRowComponent}
+                  </div>
+                  <div className="d-flex justify-content-end align-items-start flex-shrink-0">
+                  {!readsExist(this.state.tracks) && (
+                    <>
+                      <Button
+                        onClick={this.openPopup}
```

Comment:
If we rename the function we'd change it here too.
```suggestion
                        onClick={this.togglePopup}
```